package.json

```diff
@@ -2,41 +2,69 @@
     "name": "parse5-build-scripts",
     "private": true,
     "type": "module",
+    "workspaces": [
+        "packages/*",
+        "bench"
+    ],
     "devDependencies": {
+        "@types/dedent": "^0.7.0",
+        "@types/jest": "^27.4.0",
```

Comment:
How about loosening versions, e.g. `^27.0.0`, if they aren’t needed. It produces less noise in package(-lock) diffs and gives npm more room to choose what to install leading to less duplicates

Reply:
I'm mostly weary of _helpful_ PRs that bump the patch or minor version, even though these versions are already covered by the range. Instead, I'd rather keep everything up to date. Plus, there are less package versions to ask for in bug reports (not really an issue here, as all of them are dev deps).

As long as no fixed versions are used, this shouldn't make life harder for `npm`.